portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,149 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css?family=Raleway&display=swap');
+
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+  font-family: 'Raleway', sans-serif;
+}
+
+body{
+   background-color: #f3f5f9;
+}
+
+.wrapper{
+  display: flex;
+  position: relative;
+}
+
+/* -------------- Side Bar --------------*/
+
+.wrapper .sidebar{
+  width: 280px;
+  height: 100%;
+  background: #6767b5;
+  padding: 30px 0px;
+  position: fixed;
+}
+
+/* -------------- Side Bar Avatar --------------*/
+
+.wrapper .sidebar .avatar {
+    position: relative;
+    overflow: hidden;
+    margin: 55px;
+    width: 170px;
+    height: 170px;
+    -webkit-border-radius: 50%;
+    -moz-border-radius: 50%;
+    -ms-border-radius: 50%;
+    border-radius: 50%;
+}
+.wrapper .sidebar .avatar img {
+  width: 100%;
+}
+
+/* -------------- Side Bar Headings --------------*/
+
+.wrapper .sidebar h2{
+  color: #fff;
+  /* text-transform: uppercase; */
+  text-align: center;
+  margin-bottom: 30px;
+}
+
+.wrapper .sidebar h4{
+  color: #fff;
+  text-align: center;
+  margin-bottom: 30px;
+}
+
+/* -------------- Side Bar Menu --------------*/
+
+.wrapper .sidebar .menu ul li{
+  padding: 15px;
+  border-bottom: 1px solid #bdb8d7;
+  border-bottom: 1px solid rgba(0,0,0,0.05);
+  border-top: 1px solid rgba(255,255,255,0.05);
+}
+
+.wrapper .sidebar .menu ul li a{
+  color: #bdb8d7;
+  display: block;
+  padding-left: 30px;
+}
+
+.wrapper .sidebar .menu ul li a .fas{
+  width: 25px;
+}
+
+.wrapper .sidebar .menu ul li:hover{
```

Comment:
Internal Google recommendation is set a unique class for these elements you're trying to target to avoid having this long chain of selectors (which break if you refactor the HTML), but up to you.